src/editors/containers/ProblemEditor/components/EditProblemView/index.jsx

```diff
@@ -5,21 +5,31 @@ import SettingsWidget from './SettingsWidget';
 import QuestionWidget from './QuestionWidget';
 import { EditorContainer } from '../../../EditorContainer';
 import { selectors } from '../../../../data/redux';
-import { useSelector } from 'react-redux';
-import { Editor } from '@tinymce/tinymce-react';
-import { ToggleButton, ToggleButtonGroup } from '@edx/paragon';
+import { ReactStateParser } from '../../data/ReactStateParser';
+import { injectIntl } from '@edx/frontend-platform/i18n';
+import { connect } from 'react-redux';
 
-export const EditProblemView = () => {
-  const problemType = useSelector(selectors.problem.problemType);
-  const question = useSelector(selectors.problem.question);
+
+export const EditProblemView = ({
+  problemType,
+  problemState
+}) => {
+  const parseSate = (problemState) => () => {
+      let reactParser = new ReactStateParser(problemState);
+      return reactParser.getMarkdown();
+  }
   return (
-      <EditorContainer getContent={() => ({})}>
-        <Editor initialValue={question}/>
+      <EditorContainer getContent={parseSate(problemState)}>
+        <QuestionWidget />
         <AnswerWidget problemType={problemType} />
         <SettingsWidget />
-        <QuestionWidget />
       </EditorContainer>
   );
 };
 
-export default EditProblemView;
+export const mapStateToProps = (state) => ({
+  problemType: selectors.problem.problemType(state),
+  problemState: selectors.problem.completeState(state),
+})
+
+export default injectIntl(connect(mapStateToProps)(EditProblemView));
```

Comment:
`injectIntl` probably not needed here. It is mostly used if you need to inject `intl` into props.

https://github.com/open-craft/frontend-lib-content-components/blob/476c450e6c6357e34ddf7c011c6e2237345db430/src/editors/containers/TextEditor/index.jsx#L133-L149